.gitmodules

```diff
@@ -35,3 +35,6 @@
 	path = tpls/qpp
 	url = https://github.com/softwareQinc/qpp.git
 	shallow = true
+[submodule "tpls/cpr"]
+	path = tpls/cpr
+	url = https://github.com/libcpr/cpr.git
```

Comment:
[Adding comment here about the tag]
Q: Why are we using `v1.9.0` which is ~2 years old? Why not the latest stable version?

Reply:
I don't know if that decision has some background to it. @1tnguyen might know?

I was conservative here and preserved the same SHA number.